INSTALL

```diff
@@ -1,575 +1,176 @@
+ - libkf5aarchive
 
-     http://ftp.gnome.org/pub/gnome/sources/gtk+/3.22/gtk+-3.22.12.tar.xz
+ - libSDL2 Mixer
 
-   "Gtk+" depends on the "Glib", "Atk", "Gdk-Pixbuf", and "Pango" libraries.
+ - python3
 
-If you are going to make these yourself, build and install them in the
-following order: pkg-config, Glib, Atk, Pango, Gdk-Pixbuf, Gtk+.
 
-1c. Prerequisites for the SDL2 client:
-=====================================
+1. Prerequisites for the Qt client:
+===================================
 
- - The "SDL2" library.
-
-   "Simple DirectMedia Layer is a cross-platform multimedia library designed
-    to provide low level access to audio, keyboard, mouse, joystick, 3D hardware
-    via OpenGL, and 2D video framebuffer." (http://www.libsdl.org)
-
-   These features make it a good choice for portable games. You may obtain it
-   at:
-
-   http://www.libsdl.org/release/SDL2-2.0.3.tar.gz
-
-   To compile the client using the "SDL2" library add "--enable-client=sdl2"
-   to the Freeciv configure script.  See the section below for more 
-   information about the configure script.
-
- - The "SDL2_image" library.
- 
-   This library loads the PNG images (with the help of libpng) and converts
-   them to "SDL surfaces" that can be handled by the "SDL" library. You may
-   obtain it at:
-
-   http://www.libsdl.org/projects/SDL_image/release/SDL2_image-2.0.1.tar.gz
-
- - The "SDL2_gfx" library.
-
-   http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.3.tar.gz
-
- - The "SDL2_ttf" library.
-
-   http://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-2.0.14.tar.gz
-
- - The "freetype" library.
-
-   This library helps to render text for the SDL2 client, using an externally
-   provided TrueType font. You may obtain it at:
-
-   http://download.savannah.gnu.org/releases/freetype/freetype-2.7.1.tar.bz2
-
-1d. Prerequisites for the Qt client:
-====================================
+The Freeciv21 project maintains a single Qt client.
 
  - C++ compiler.
 
-   Qt-client is written in C++, so you need appropriate compiler.
-   In Freeciv development, g++ has been used.
-
- - Qt5Core, Qt5Gui, and Qt5Widgets libraries and headers.
-   At least version 5.11 is required.
-
-
-2. Generating Makefiles
-=======================
-This section contains two parts, one for generating makefiles from git
-versions and one for generating makefiles from release versions.
+Qt-client is written in C++, so you need appropriate compiler. In Freeciv21 development, g++ has been used.
 
-2a. Generating the Makefile for git versions:
-=============================================
+ - Qt5Core, Qt5Gui, and Qt5Widgets libraries and headers. At least version 5.11 is required.
 
-This step is only needed for git versions.
 
-To create the makefile just type
+2. Compiling and Installing Freeciv21:
+======================================
 
- % ./autogen.sh
+Compiling Freeciv21 requires the use of the cmake program. cmake does require the use of command line parameters to function
```

Comment:
Actually one just needs to type `cmake . -B build` to create a default (unoptimized) `make`-based build dir.

Reply:
Fixed in third commit